### PR TITLE
Project configurations modified

### PR DESCRIPTION
**Pull request by deshraj**
- dev.py added in project
- Accounts app deleted(not needed)
- Basic "hello world" app created so that angular integration will become easy
- All Apps added in INSTALLED_APPS
- Autopep8 added in Contribution Guidelines

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -51,7 +51,6 @@ coverage.xml
 
 # Django stuff:
 *.log
-dev.py
```
Comment (dexter1691) [inline .gitignore]: Already taken care of in #14 ?

Comment (deshraj) [inline .gitignore]: Won't be an issue I think.

File: settings/dev.py [hunk 2]
```diff
@@ -9,11 +9,11 @@
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
         'NAME': 'evalai',
-        'USER': '',
-        'PASSWORD': '',
+        'USER': 'deshraj',
```
Comment (dexter1691) [inline settings/dev.py]: hardcoded values found! 🗡 💀

File: settings/dev.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+# Database
+# https://docs.djangoproject.com/en/1.10.2/ref/settings/#databases
+
+DEBUG = True
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'evalai',
+        'USER': 'deshraj',
```
Comment (dexter1691) [inline settings/dev.py]: hardcoded values found 🗡 💀

**Comment by dexter1691** [comment]
@deshraj : Can you merge the latest master and resolve the conflict.

**Comment by deshraj** [comment]
Sure.

**Pull request merged**